### PR TITLE
input/dns_query move defaults to init()

### PR DESCRIPTION
**Pull request by tmonk42**
### Required for all PRs:

- [ ] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] README.md updated (if adding a new plugin)

### CODE REVIEW

**Comment by tmonk42** [comment]
Following the pattern in #1673

**Comment by sparrc** [comment]
@tmonk42 why did you close this?

**Comment by tmonk42** [comment]
@sparrc I couldn't figure out the correct way to test the defaults in dns_query_test.go. I'm a Go newbie and figured I'd take a crack at one of the other plugins while I was on a roll and got in over my head. I'm still trying to figure out how to get my Go development and testing environment setup so I don't depend upon CircleCI to do testing for my proposed changes, lol. If testing the default values isn't required, I could yank that bit out of the test file and rest of it should be good-to-go.

File: plugins/inputs/dns_query/dns_query.go [hunk 1]
```diff
@@ -157,6 +136,11 @@ func (d *DnsQuery) parseRecordType() (uint16, error) {
 
 func init() {
 	inputs.Add("dns_query", func() telegraf.Input {
-		return &DnsQuery{}
+		return &DnsQuery{
```
Comment (sparrc) [inline plugins/inputs/dns_query/dns_query.go]: you could create a function called `NewDnsQuery` which returns this struct. Then you can call NewDnsQuery() from within init() _and_ from the unit tests.

**Comment by sparrc** [comment]
no problem, see my comment about making a "New" function, let me know if you have any other questions

**Comment by sparrc** [comment]
closing for lack of activity, feel free to open a new PR if you can make the requested changes

**Pull request closed without merging**